clusters/build-clusters/common/ci-operator/admin_ci-operator_rbac.yaml

```diff
@@ -302,6 +302,32 @@ items:
     name: ci-operator
     namespace: ci
 
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: ci-secret-getter
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - secrets
+      verbs:
+        - get
+        - list
+
+- kind: ClusterRoleBinding
```

Comment:
This will allow the `ci-operator` service account to get/list secrets from all namespaces. Instead, we need to restrict it to the ci namespace.